.ebextensions/cronjob.config

```diff
@@ -95,54 +73,58 @@ files:
         content: |
             45 8 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bundle exec rake amr_importer:send_daily_notification_email >> log/jobs.log 2>&1'
     #
-    # Weekly alerts
-    #
-    "/etc/cron.d/weekly-alerts":
-        mode: "000644"
-        owner: root
-        group: root
-        content: |
-            0 8 * * Wed root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bundle exec rake alerts:create_weekly >> log/jobs.log 2>&1'
-    #
-    # Holiday alerts
+    # Generate alerts daily
     #
-    "/etc/cron.d/holiday-alerts":
+    "/etc/cron.d/daily-alert-generation":
```

Comment:
I think a pre-requisite for this task should be clearing of the cache. Or, we should do it after validation. If the following tasks (i.e. find out mores) use a cached aggregated school that doesn't match the current live validated data then we might be out of sync again.